cdap-client-tests/src/test/java/co/cask/cdap/client/ArtifactClientTestRun.java

```diff
@@ -82,6 +81,10 @@ public InputStream getInput() throws IOException {
   public void setUp() throws Throwable {
     super.setUp();
     artifactClient = new ArtifactClient(clientConfig, new RESTClient(clientConfig));
+    for (ArtifactSummary artifactSummary : artifactClient.list(Id.Namespace.DEFAULT, false)) {
```

Comment:
Do we always expect some artifacts to be there (so that ``artifactClient.delete`` gets invoked) ?  Or should we add an explicit call in the test and check that the artifact has been deleted?